### ISSUE TITLE
spdlog_devel requires libfmt_devel but doesn't depend on it

### ISSUE THREAD

**Issue by kallisti5** [haikuports/haikuports#5656]
```
 pkg-config spdlog --libs
Package fmt was not found in the pkg-config search path.
Perhaps you should add the directory containing `fmt.pc'
to the PKG_CONFIG_PATH environment variable
Package 'fmt', required by 'spdlog', not found
```

```
    install package libfmt_devel-7.0.3-1 from repository HaikuPorts
```

```
 pkg-config spdlog --libs
-L/packages/spdlog-1.8.0-1/.self/develop/lib -L/packages/libfmt-7.0.3-1/.self/develop/lib -lspdlog -pthread -lfmt
```

**Comment by korli** [comment]
Should be fixed in 612ab38abe0a597d665e223ea806af5de6b72c01   or 1.8.0-2

**Comment by kallisti5** [comment]
Thanks!
